bench/setup_sklearn_bench.sh

```diff
@@ -9,10 +9,10 @@ trap "kill 0" EXIT
 
 cifar_path=$1
 
-python bench_init.py $cifar_path
+python bench/bench_init.py $cifar_path
 
 export CLIPPER_MODEL_NAME="bench_sklearn_cifar"
 export CLIPPER_MODEL_VERSION="1"
 export CLIPPER_MODEL_PATH="model/"
 
-python ../containers/python/sklearn_cifar_container.py 
+python containers/python/sklearn_cifar_container.py 
```

Comment:
In `end_to_end_bench`, we prompt the user to run this script (`bench/setup_sklearn_bench.sh`) from the root directory. This change lets it work from the root directory (before, it would work only if run from within `bench/`).